### PR TITLE
LPS-109844 Create drafts of structured content

### PR DESCRIPTION
**Pull request by LuismiBarcos**

### CODE REVIEW

**Comment by LuismiBarcos** [comment]
I've made two principal changes:
* Change the name of the package com.liferay.headless.delivery.util to com.liferay.headless.delivery.util.v1_0 to take into account the versioning and avoid future problems as Brian said.
* Change the name of the exception mapper DDMFormValuesValidationExceptionMapper to InvalidDDMFormValueExceptionMapper since Brian says that: `Remember, exception mappers are tied to the REST name and not the Service Builder name. Please resend.`

**Comment by nhpatt** [comment]
@LuismiBarcos the comment about exception mapper also refers means that `DMFormValue` does not exist outside liferay and in the APIs is called differently, `StructuredContent`. So the name should be something like `StructuredContentContentFieldsExceptionMapper`

File: modules/apps/headless/headless-admin-content/headless-admin-content-impl/src/main/java/com/liferay/headless/admin/content/internal/jaxrs/exception/mapper/InvalidDDMFormValueExceptionMapper.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Admin.Content)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Headless.Admin.Content.InvalidDDMFormValueExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class InvalidDDMFormValueExceptionMapper
```
Comment (nhpatt) [inline modules/apps/headless/headless-admin-content/headless-admin-content-impl/src/main/java/com/liferay/headless/admin/content/internal/jaxrs/exception/mapper/InvalidDDMFormValueExceptionMapper.java:39]: DDMFormValue is an internal name

**Review by nhpatt — CHANGES_REQUESTED** [review]

**Comment by nhpatt** [comment]
ci:forward

**Review by nhpatt — APPROVED** [review]

**Comment by LuismiBarcos** [comment]
ci:forward

**Pull request closed without merging**